config/releases.exs

```diff
@@ -1 +1,98 @@
 import Config
+
```

Comment:
we can have a portion here that sets the`:single_tenant` config key, something like
```
config :logflare, single_tenant: System.get_env("LOGFLARE_SINGLE_TENANT")
```
which will get merged with the compile-time keys, so the final config on `:logflare` is
```
# for prod single tenant
config :logflare, env: :prod, single_tenant: true


# for dev single tenant
config :logflare, env: :dev, single_tenant: true

# for dev cluster
config :logflare, env: :dev, single_tenant: false

# for prod cluster
config :logflare, env: :prod, single_tenant: false
```
we should also think of phasing out the `staging` env, because we should see it as a "prod" environment with its own env var file, so then our staging will be as close to prod as possible.

Reply:
This sounds nice. 

Let's move away from "single tenant" because frankly I'd rather not feature flag a bunch of stuff just to feature flag it. The features are there so let them use them if they want to figure it out. If they're motivated enough they'll fork it and figure it out anyways. 

Maybe "self host".